frontend/src/features/public-form/components/FormFields/FormFieldsContainer.tsx

```diff
@@ -40,11 +50,41 @@ export const FormFieldsContainer = (): JSX.Element | null => {
     )
   }, [form, handleSubmitForm, isAuthRequired, isLoading])
 
+  // TODO #4279: Computes whether to show the switch env message, remove after React rollout is complete
```

Comment:
should we move this into publicswitchenvmessage component? This is sticking around longer than we thought lol.

the component itself can control whether or not it is shown. can pass in form response type as a prop.